### DISCUSSION TITLE
Interested in being a GSoC 2024 contributor

### DISCUSSION THREAD

**Discussion by SAKSHAM0002** [robolectric/robolectric#8900] [General]
Hello,
I am saksham,a 2nd year undergraduate.I am deeply passionate about learning and contributing to meaningful projects but I do not possess extensive technical knowledge.I believe that GSoC provides a fantastic opportunity for me to immerse myself in the world of open-source development, collaborate with experienced mentors,and make a positive impact on projects that align with my interests.
Could you please guide me in this.

**Comment by utzcoz** [comment]
@saxamm Glad to know you're interested in Robolectric' GSoC 2024 projects, and you can check https://robolectric.org/blog/2024/02/25/robolectric-2024-gsoc/ to learn what does Robolectric expect for potential contributors. Also https://github.com/robolectric/robolectric/discussions/8865 provides some tips to start.

**Comment by SAKSHAM0002** [comment]
My exams just got over and I have not been able to make any contributions
to this community, nor have I proposed any project ideas, which concludes
on the 18th of March. Unfortunately, it is very unlikely that I will be
able to propose anything before this deadline.

I was really interested in being a contributor but I also currently lack
the required technical knowledge for being a potential candidate.
Depending on my situation, could you please guide me if I still have a
chance of becoming a potential candidate if I start contributing now or I
should just start acquiring knowledge from now on and try again next year.

Your reply would be really helpful for me.

Thank You

On Sun, Mar 10, 2024 at 3:05 PM utzcoz ***@***.***> wrote:

> @saxamm <https://github.com/saxamm> Glad to know you're interested in
> Robolectric' GSoC 2024 projects, and you can check
> https://robolectric.org/blog/2024/02/25/robolectric-2024-gsoc/ to learn
> what does Robolectric expect for potential contributors. Also #8865
> <https://github.com/robolectric/robolectric/discussions/8865> provides
> some tips to start.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/robolectric/robolectric/discussions/8900#discussioncomment-8734631>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGMYU5NXQSXZHFYSKPI643DYXQSPNAVCNFSM6AAAAABEOWHS3WVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DOMZUGYZTC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

-- 
The information contained in this electronic communication is intended 
solely for the individual(s) or entity to which it is addressed. It may 
contain proprietary, confidential and/or legally privileged information. 
Any review, retransmission, dissemination, printing, copying or other use 
of, or taking any action in reliance on the contents of this information by 
person(s) or entities other than the intended recipient is strictly 
prohibited and may be unlawful. If you have received this communication in 
error, please notify us by responding to this email or telephone and 
immediately and permanently delete all copies of this message and any 
attachments from your system(s). The contents of this message do not 
necessarily represent the views or policies of BITS Pilani.

**Reply by utzcoz** [reply]
You can check  https://robolectric.org/blog/2024/02/25/robolectric-2024-gsoc/  for our expectation for potential contributors.

**Comment by harshal-rembhotkar** [comment]
Hello ,

I am Harshal Rembhotkar, a 4th-semester Engineering student. I am excited to join your organization. My tech stacks are Java, HTML, CSS, version control systems, and technologies such as Docker and Kubernetes.I am completely enthusiastic about taking part in GSOC 2024 in Robolectric and giving my best in it!
Thank you..

Linkedin: https://www.linkedin.com/in/harshal-rembhotkar/
Github: https://github.com/Harshal-Rembhotkar

**Reply by utzcoz** [reply]
@Harshal-Rembhotkar Glad to know you're interested in Robolectric' GSoC 2024 projects, and you can check https://robolectric.org/blog/2024/02/25/robolectric-2024-gsoc/ to learn what does Robolectric expect for potential contributors. Also https://github.com/robolectric/robolectric/discussions/8865 provides some tips to start.
